src/main/webapp/app/view/project/screenplay/screenplay.component.html

```diff
@@ -1,6 +1,43 @@
 <div>
     <h2 id="page-heading">
-        <span jhiTranslate="eventPlannerApp.project.screenplay.title">Screenplay</span>
-        <i>{{project?.name}}</i>
+        <span>{{project?.name}}</span>
```

Comment:
Evtl. könnten wir im Titel jeweils noch schreiben ob wir im Drehbuch oder Einstellungen sind. z.B Turnfest Frutigen Drehbuch. Könnten wir in der nächsten Sitzung mit Thomas anschauen.

Reply:
finde ich ok, habe ich hinzugefügt.

Reply:
Hast du die Anpassungen gepusht?